**Windows** [Build 5182](https://ci.wildfly.org/viewLog.html?buildId=23750&buildTypeId=WF_PullRequest_Windows) outcome was **FAILURE** using a merge of 744b8eaad023e6ee691e931823a96014624d0031
Summary: Tests failed: 1 (1 new), passed: 3424, ignored: 194 Build time: 01:24:37
### Failed tests
```
org.jboss.as.test.clustering.cluster.web.ConcurrentCoarseWebFailoverTestCase(SYNC-tcp).testGracefulUndeployFailover: java.lang.AssertionError: Session failed to replicate after container 1 was shutdown. expected:<3> but was:<2>
	at org.jboss.as.test.clustering.cluster.web.AbstractWebFailoverTestCase.testFailover(AbstractWebFailoverTestCase.java:143)
	at org.jboss.as.test.clustering.cluster.web.AbstractWebFailoverTestCase.testGracefulUndeployFailover(AbstractWebFailoverTestCase.java:96)
------- Stdout: -------
00:15:42,215 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-0
00:15:42,216 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Starting deployment=NONE, container=container-1
00:15:42,216 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-0
00:15:42,244 INFO  [org.jboss.as.repository] (management-handler-thread - 3) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-0\standalone\data\content\78\492a71bf711ea49dc7cd601e727db13897232a\content
00:15:42,247 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0027: Starting deployment of "coarse-concurrent-distributable.war" (runtime-name: "coarse-concurrent-distributable.war")
00:15:42,287 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-8) WFLYSRV0018: Deployment "deployment.coarse-concurrent-distributable.war" is using a private module ("org.jboss.as.clustering.common:main") which may be changed or removed in future versions without notice.
00:15:42,287 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-8) WFLYSRV0018: Deployment "deployment.coarse-concurrent-distributable.war" is using a private module ("org.jboss.as.server:main") which may be changed or removed in future versions without notice.
00:15:42,287 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-8) WFLYSRV0018: Deployment "deployment.coarse-concurrent-distributable.war" is using a private module ("org.infinispan:main") which may be changed or removed in future versions without notice.
00:15:42,293 INFO  [org.jboss.weld.deployer] (MSC service thread 1-2) WFLYWELD0003: Processing weld deployment coarse-concurrent-distributable.war
00:15:42,303 INFO  [org.jboss.as.ejb3.deployment] (MSC service thread 1-2) WFLYEJB0473: JNDI bindings for session bean named 'TopologyChangeListenerBean' in deployment unit 'deployment "coarse-concurrent-distributable.war"' are as follows:

	java:global/coarse-concurrent-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:app/coarse-concurrent-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:module/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:jboss/exported/coarse-concurrent-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:global/coarse-concurrent-distributable/TopologyChangeListenerBean
	java:app/coarse-concurrent-distributable/TopologyChangeListenerBean
	java:module/TopologyChangeListenerBean

00:15:42,333 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 11) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:15:42,334 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 11) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:15:42,350 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 11) WFLYCLINF0002: Started client-mappings cache from ejb container
00:15:42,352 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 26) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:15:42,352 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 26) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:15:42,352 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:15:42,353 INFO  [org.infinispan.configuration.cache.EvictionConfigurationBuilder] (ServerService Thread Pool -- 35) ISPN000152: Passivation configured without an eviction policy being selected. Only manually evicted entities will be passivated.
00:15:42,360 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 26) WFLYCLINF0002: Started routing cache from web container
00:15:42,390 INFO  [org.jboss.as.clustering.infinispan] (ServerService Thread Pool -- 11) WFLYCLINF0002: Started coarse-concurrent-distributable.war cache from web container
00:15:42,582 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 11) WFLYUT0021: Registered web context: /coarse-concurrent-distributable
00:15:42,607 INFO  [org.jboss.as.server] (management-handler-thread - 3) WFLYSRV0010: Deployed "coarse-concurrent-distributable.war" (runtime-name : "coarse-concurrent-distributable.war")
00:15:42,922 WARN  [org.jboss.remotingjmx.Util] (main) The protocol 'http-remoting-jmx' is deprecated, instead you should use 'remote+http'.
00:15:42,978 INFO  [org.jboss.as.test.clustering.NodeUtil] (main) Deploying deployment=deployment-1
00:15:43,011 INFO  [org.jboss.as.repository] (management-handler-thread - 2) WFLYDR0001: Content added at location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\clustering\target\wildfly-SYNC-tcp-1\standalone\data\content\78\492a71bf711ea49dc7cd601e727db13897232a\content
00:15:43,015 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-8) WFLYSRV0027: Starting deployment of "coarse-concurrent-distributable.war" (runtime-name: "coarse-concurrent-distributable.war")
00:15:43,065 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-7) WFLYSRV0018: Deployment "deployment.coarse-concurrent-distributable.war" is using a private module ("org.jboss.as.clustering.common:main") which may be changed or removed in future versions without notice.
00:15:43,065 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-7) WFLYSRV0018: Deployment "deployment.coarse-concurrent-distributable.war" is using a private module ("org.jboss.as.server:main") which may be changed or removed in future versions without notice.
00:15:43,066 WARN  [org.jboss.as.dependency.private] (MSC service thread 1-7) WFLYSRV0018: Deployment "deployment.coarse-concurrent-distributable.war" is using a private module ("org.infinispan:main") which may be changed or removed in future versions without notice.
00:15:43,073 INFO  [org.jboss.weld.deployer] (MSC service thread 1-6) WFLYWELD0003: Processing weld deployment coarse-concurrent-distributable.war
00:15:43,088 INFO  [org.jboss.as.ejb3.deployment] (MSC service thread 1-6) WFLYEJB0473: JNDI bindings for session bean named 'TopologyChangeListenerBean' in deployment unit 'deployment "coarse-concurrent-distributable.war"' are as follows:

	java:global/coarse-concurrent-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:app/coarse-concurrent-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:module/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:jboss/exported/coarse-concurrent-distributable/TopologyChangeListenerBean!org.jboss.as.test.clustering.TopologyChangeListener
	java:global/coarse-concurrent-distributable/TopologyChangeListenerBean
	java:app/coarse-concurrent-distributable/TopologyChangeListenerBean
	java:module/TopologyChangeListenerBean


```

--- a/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/passivation/ClusterPassivationTestCase.java
+++ b/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/passivation/ClusterPassivationTestCase.java
@@ -43,7 +43,6 @@ import org.jboss.logging.Logger;
 import org.jboss.shrinkwrap.api.Archive;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
@@ -108,7 +107,7 @@ public class ClusterPassivationTestCase extends ClusterPassivationTestBase {
     }
 
     /**
-     * Associtation of node names to deployment,container names and client context
+     * Association of node names to deployment,container names and client context
      */
     @Test
     @InSequence(-1)
@@ -129,7 +128,6 @@ public class ClusterPassivationTestCase extends ClusterPassivationTestBase {
         log.info("URL2 nodename: " + nodeName2);
     }
 
-    @Ignore("JBPAPP-8774")
     @Test
     @InSequence(1)
     public void testPassivationBeanAnnotated(

--- a/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/CounterBaseBean.java
+++ b/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/CounterBaseBean.java
@@ -1,4 +1,29 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2012, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
 package org.jboss.as.test.clustering.extended.ejb2.stateful.remote.failover;
+
+import java.rmi.RemoteException;
+
+import javax.ejb.CreateException;
 
 import org.jboss.as.test.clustering.NodeNameGetter;
 
@@ -8,7 +33,7 @@ import org.jboss.as.test.clustering.NodeNameGetter;
 public abstract class CounterBaseBean {
     private int count;
 
-    public void ejbCreate() throws java.rmi.RemoteException, javax.ejb.CreateException {
+    public void ejbCreate() throws RemoteException, CreateException {
         // Creating method for home interface...
     }
 

--- a/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/RemoteEJBClientStatefulBeanFailoverTestCase.java
+++ b/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/RemoteEJBClientStatefulBeanFailoverTestCase.java
@@ -26,7 +26,6 @@ import org.jboss.arquillian.container.test.api.Deployment;
 import org.jboss.arquillian.container.test.api.RunAsClient;
 import org.jboss.arquillian.container.test.api.TargetsContainer;
 import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.arquillian.junit.InSequence;
 import org.jboss.as.test.clustering.NodeNameGetter;
 import org.jboss.shrinkwrap.api.Archive;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
@@ -81,14 +80,12 @@ public class RemoteEJBClientStatefulBeanFailoverTestCase extends RemoteEJBClient
     }
 
     @Override
-    @InSequence(1)
     @Test
     public void testFailoverFromRemoteClientWhenOneNodeGoesDown() throws Exception {
         failoverFromRemoteClient(false);
     }
 
     @Override
-    @InSequence(2)
     @Test
     public void testFailoverFromRemoteClientWhenOneNodeUndeploys() throws Exception {
         failoverFromRemoteClient(true);

--- a/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/RemoteEJBClientStatefulFailoverTestBase.java
+++ b/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/RemoteEJBClientStatefulFailoverTestBase.java
@@ -1,3 +1,25 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2012, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
 package org.jboss.as.test.clustering.extended.ejb2.stateful.remote.failover;
 
 import javax.naming.NamingException;

--- a/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/dd/CounterBeanDD.java
+++ b/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/dd/CounterBeanDD.java
@@ -40,6 +40,12 @@ public class CounterBeanDD extends CounterBaseBean implements SessionBean {
     private static final long serialVersionUID = 1L;
 
     @Override
+    public void ejbRemove() throws EJBException, RemoteException {
+        log.info("ejbRemove() was called..");
+        CounterSingleton.destroyCounter.incrementAndGet();
+    }
+
+    @Override
     public void ejbActivate() throws EJBException, RemoteException {
 
     }
@@ -47,12 +53,6 @@ public class CounterBeanDD extends CounterBaseBean implements SessionBean {
     @Override
     public void ejbPassivate() throws EJBException, RemoteException {
 
-    }
-
-    @Override
-    public void ejbRemove() throws EJBException, RemoteException {
-        log.info("ejbRemove() was called..");
-        CounterSingleton.destroyCounter.intValue();
     }
 
     @Override

--- a/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/dd/RemoteEJB2ClientStatefulBeanFailoverDDTestCase.java
+++ b/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/dd/RemoteEJB2ClientStatefulBeanFailoverDDTestCase.java
@@ -26,19 +26,16 @@ import org.jboss.arquillian.container.test.api.Deployment;
 import org.jboss.arquillian.container.test.api.RunAsClient;
 import org.jboss.arquillian.container.test.api.TargetsContainer;
 import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.arquillian.junit.InSequence;
 import org.jboss.as.test.clustering.NodeNameGetter;
 import org.jboss.as.test.clustering.extended.ejb2.stateful.remote.failover.CounterBaseBean;
 import org.jboss.as.test.clustering.extended.ejb2.stateful.remote.failover.CounterRemote;
 import org.jboss.as.test.clustering.extended.ejb2.stateful.remote.failover.CounterRemoteHome;
 import org.jboss.as.test.clustering.extended.ejb2.stateful.remote.failover.CounterResult;
 import org.jboss.as.test.clustering.extended.ejb2.stateful.remote.failover.RemoteEJBClientStatefulFailoverTestBase;
-import org.jboss.logging.Logger;
 import org.jboss.shrinkwrap.api.Archive;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.asset.StringAsset;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
@@ -48,7 +45,6 @@ import org.junit.runner.RunWith;
 @RunWith(Arquillian.class)
 @RunAsClient
 public class RemoteEJB2ClientStatefulBeanFailoverDDTestCase extends RemoteEJBClientStatefulFailoverTestBase {
-    private static final Logger log = Logger.getLogger(RemoteEJB2ClientStatefulBeanFailoverDDTestCase.class);
 
     @Deployment(name = DEPLOYMENT_HELPER_1, managed = false, testable = false)
     @TargetsContainer(CONTAINER_1)
@@ -80,21 +76,16 @@ public class RemoteEJB2ClientStatefulBeanFailoverDDTestCase extends RemoteEJBCli
         jar.addClass(NodeNameGetter.class);
         jar.addAsManifestResource(RemoteEJB2ClientStatefulBeanFailoverDDTestCase.class.getPackage(), "ejb-jar.xml", "ejb-jar.xml");
         jar.addAsManifestResource(new StringAsset("Dependencies: deployment." + ARCHIVE_NAME_SINGLE + ".jar\n"), "MANIFEST.MF");
-        log.info(jar.toString(true));
         return jar;
     }
 
-    @Ignore("JBPAPP-8726")
     @Override
-    @InSequence(1)
     @Test
     public void testFailoverFromRemoteClientWhenOneNodeGoesDown() throws Exception {
         failoverFromRemoteClient(false);
     }
 
-    @Ignore("JBPAPP-8726")
     @Override
-    @InSequence(2)
     @Test
     public void testFailoverFromRemoteClientWhenOneNodeUndeploys() throws Exception {
         failoverFromRemoteClient(true);

--- a/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/dd/ejb-jar.xml
+++ b/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/extended/ejb2/stateful/remote/failover/dd/ejb-jar.xml
@@ -32,13 +32,5 @@
             <session-type>Stateful</session-type>
             <transaction-type>Container</transaction-type>
         </session>
-        <session>
-            <ejb-name>ViewChangeListenerBean</ejb-name>
-            <home>org.jboss.as.test.clustering.ViewChangeListenerHome</home>
-            <remote>org.jboss.as.test.clustering.ViewChangeListener</remote>
-            <ejb-class>org.jboss.as.test.clustering.ViewChangeListenerBean</ejb-class>
-            <session-type>Stateless</session-type>
-            <transaction-type>Bean</transaction-type>
-        </session>
     </enterprise-beans>
 </ejb-jar>

--- a/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/xsite/XSiteSimpleTestCase.java
+++ b/testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/xsite/XSiteSimpleTestCase.java
@@ -46,7 +46,6 @@ import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.asset.StringAsset;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.junit.Assert;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
@@ -115,13 +114,11 @@ public class XSiteSimpleTestCase extends ExtendedClusterAbstractTestCase {
      *   arrives at SFO-0 on site SFO
      */
     @Test
-    @Ignore("https://issues.jboss.org/browse/WFLY-5239")
     public void testPutRelayedToBackups(
             @ArquillianResource(CacheAccessServlet.class) @OperateOnDeployment(DEPLOYMENT_1) URL baseURL1,
             @ArquillianResource(CacheAccessServlet.class) @OperateOnDeployment(DEPLOYMENT_2) URL baseURL2,
             @ArquillianResource(CacheAccessServlet.class) @OperateOnDeployment(DEPLOYMENT_3) URL baseURL3,
             @ArquillianResource(CacheAccessServlet.class) @OperateOnDeployment(DEPLOYMENT_4) URL baseURL4)
-
             throws IllegalStateException, IOException, URISyntaxException {
 
         String value = "100";

